protocol/contracts/dao/Regulator.sol

```diff
@@ -106,6 +106,6 @@ contract Regulator is Comptroller {
         }
 
         // Note: Price comes in x 1e12, normalize here
-        return price.div(1e12);
+        return price.div(1e12).mul(4);
```

Comment:
so this line defines the repeg? no multipliers on the debt or supply required? (the next commit removes the changes on lines 57 & 68)

Reply:
Correct. Because `step()` calls into `oracleCapture()` which now repegs the price, that repegged price trickles down into `growSupply()` and `shrinkSupply()`.